src/apex-stubs/apex/apex.js

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+export const getSObjectValue = jest.fn();
```

Comment:
How does this relate to https://github.com/salesforce/lwc-test/blob/master/packages/%40lwc/jest-transformer/src/transforms/apex-scoped-import.js#L16-L72 ?

We do already transform those `apex` imports.